banzai_nres/frames.py

```diff
@@ -405,4 +405,5 @@ def open(self, path, runtime_context) -> Optional[ObservationFrame]:
                 # Convert to mas / yr from arcsec / year
                 image.pm_ra = image[f'TELESCOPE_{telescope_num}'].meta['PM-RA'] * 1000.0
                 image.pm_dec = image[f'TELESCOPE_{telescope_num}'].meta['PM-DEC'] * 1000.0
+            image.meta['RADESYS'] = image[f'TELESCOPE_{telescope_num}'].meta['RADECSYS']
```

Comment:
What is the difference between 'RADESYS' and 'RADECSYS'?

Reply:
Typo.